api/v1alpha1/provisioning_types.go

```diff
@@ -227,6 +227,9 @@ type ProvisioningSpec struct {
 	// ironic-prometheus-exporter container, and creates supporting resources
 	// (ServiceMonitor, Service ports) to expose hardware sensor metrics for Prometheus.
 	PrometheusExporter *PrometheusExporter `json:"prometheusExporter,omitempty"`
+
+	// IronicExternalIP is the external-facing IP address used to access the Ironic service.
```

Comment:
We probably want to avoid "Ironic" in the name, especially since it's not just Ironic.